src/fsharp/vs/service.fs

```diff
@@ -505,6 +502,164 @@ type SemanticClassificationType =
     | Operator
     | Disposable
 
+/// An intellisense declaration
```

Comment:
Is it absolutely critical that this be moved from ServiceDeclarations.fs?  It makes the diff a lot larger.  

As mentioned below we really need to make this diff as minimal as possible textually to avoid conflicts when we merge into FSharp.Compiler.Service.dll, where there are still significant changes in service.fs